### PR TITLE
changed BetterSettlers order according to requirements

### PR DESCRIPTION
**Pull request by soyasoya5**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -2336,15 +2336,20 @@ plugins:
     url: [ 'http://www.nexusmods.com/fallout4/mods/4772' ]
     after:
       - 'MoreSkinColors.esp'
+      - 'AzarPonyHairstyles.esp'
+      - 'Eli_Armour_Compendium.esp'
+      - 'Scrap Everything.+\.esp'
```
Comment (Ortham) [inline masterlist.yaml]: After entries need to be exact plugin names, that regex (and the others in your commit) won't be used as a regex.

File: masterlist.yaml [hunk 2]
```diff
@@ -2336,15 +2336,20 @@ plugins:
...
         util: 'FO4Edit v3.2'
   - name: 'BetterSettlersFarHarborPatchMortal.esp'
     url: [ 'http://www.nexusmods.com/fallout4/mods/4772' ]
+    req:
+      - 'BetterSettlers.esp'
     after:
       - 'BetterSettlersNoLollygagging.esp'
       - 'BetterSettlersCleanFacePack.esp'
-      - 'BetterSettlersCCAPack1.1b.esp'
+      - 'BetterSettlersCCAPack.+\.esp'
```
Comment (Ortham) [inline masterlist.yaml]: See my earlier comment, you'll need to list the exact plugins instead.

File: masterlist.yaml [hunk 3]
```diff
@@ -2353,16 +2358,25 @@ plugins:
     inc: [ 'BetterSettlersFarHarborPatch.esp' ]
   - name: 'BetterSettlersFarHarborPatch.esp'
     url: [ 'http://www.nexusmods.com/fallout4/mods/4772' ]
+    req:
+      - 'BetterSettlers.esp'
     after:
       - 'BetterSettlersNoLollygagging.esp'
       - 'BetterSettlersCleanFacePack.esp'
-      - 'BetterSettlersCCAPack1.1b.esp'
+      - 'BetterSettlersCCAPack.+\.esp'
```
Comment (Ortham) [inline masterlist.yaml]: See my earlier comment, you'll need to list the exact plugins instead.

**Review by Ortham — CHANGES_REQUESTED** [review]
Thanks for contributing, I've commented on a few issues.

**Comment by soyasoya5** [comment]
Hi, thanks for the comments. I've made the requested changes and squashed commits.

**Review by Ortham — APPROVED** [review]

**Comment by Ortham** [comment]
Thanks!

**Pull request merged**